compositor_pipeline/src/pipeline/input/rtp/depayloader.rs

```diff
@@ -0,0 +1,119 @@
+use rtp::{
+    codecs::{h264::H264Packet, opus::OpusPacket},
+    packetizer::Depacketizer,
+};
+
+use crate::pipeline::structs::{AudioCodec, EncodedChunk, EncodedChunkKind, VideoCodec};
+
+use super::{DepayloadingError, RtpStream};
+
+pub struct Depayloader {
+    /// (Depayloader, payload type)
+    pub video: Option<(VideoDepayloader, u8)>,
+    pub audio: Option<(AudioDepayloader, u8)>,
```

Comment:
I didn't want to create another struct for that, because naming it would be nightmare

Reply:
Firstly I thought about sth like:
```rust
struct SomeName { 
     depayloader: VideoDepayloader,
     payload_type: u8
}
```
because this `u8` isn't self-explanatory. After reading your comment, I realized, that using `PayloadType(u8)` would be.

I don't want to use separate fields here, because that doesn't encapsulate in type system information, that both of those values have to be the same kind (either both `None` or both `Some`).